src/Symfony/Component/Translation/Dumper/YamlFileDumper.php

```diff
@@ -24,9 +25,19 @@ class YamlFileDumper extends FileDumper
     /**
      * {@inheritdoc}
      */
-    protected function format(MessageCatalogue $messages, $domain)
+    protected function formatCatalogue(MessageCatalogue $messages, $domain, array $options = array())
     {
-        return Yaml::dump($messages->all($domain));
+        $data = $messages->all($domain);
+
+        if (isset($options['as_tree']) && $options['as_tree']) {
+            $data = ArrayConverter::expandToTree($data);
+        }
+
+        if (isset($options['inline']) && ($inline = (int) $options['inline']) > 0) {
+            return Yaml::dump($data, $inline);
+        }
+
+        return Yaml::dump($data);
```

Comment:
Yaml:dump second parameter is integer with default value 2 - "The level where you switch to inline YAML".

So, it can't be simplified neither like this nor
```
Yaml::dump($data, isset($options['inline']) ? (int)$options['inline'] : 2);
```
because default value for argument can be changed.